app/src/docs/examples/dropdown/CloseOpenDelayModifiers.example.tsx

```diff
@@ -0,0 +1,169 @@
+                    />
+
+                    <FlexCell width='100%'>
+                        <Text cx={ css.text } lineHeight='24' fontSize='16' color='gray80' font='sans-semibold'>John Doe</Text>
+                        <Text cx={ css.text } lineHeight='18' fontSize='12' color='gray60'>Corporate Function Management | L3</Text>
+                    </FlexCell>
+                </FlexRow>
+            </DropdownContainer>
+        );
+    };
+
+    return (
+        <div>
+            <FlexRow vPadding='12' spacing='12' >
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+            </FlexRow>
+            <FlexRow vPadding='12' spacing='12' >
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+                <Dropdown
+                    renderBody={ (props) => renderDropdownBody(props) }
+                    renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
+                    openOnHover={ true }
+                    openDelay={ 500 }
+                    closeOnMouseLeave='toggler'
+                />
+            </FlexRow>
+            <FlexRow vPadding='12' spacing='12' >
+                <Dropdown
```

Comment:
it seems like all 3 elements are identical. Not a big deal, but I would propose to render it like this to avoid code dublication:
```

{
Array(3).fill(null).map((e, i) => (
            <Dropdown key={ i }  renderBody={ (props) => renderDropdownBody(props) }
                renderTarget={ (props: IDropdownToggler) => <Button caption="Hover to open (open delay)" { ...props } /> }
                openOnHover={ true } openDelay={ 500 } closeOnMouseLeave='toggler'
            />
        ))
}

```